rust/processor/src/db/common/models/default_models/parquet_write_set_changes.rs

```diff
@@ -185,7 +204,7 @@ impl WriteSetChange {
                     block_height,
                     block_timestamp,
                 );
-                (
+                return Ok(Some((
```

Comment:
Nit, the explicit return is not necessary here, I'm surprised clippy didn't complain, I guess we don't have that check configured.